src/staticanalysis/bot/static_analysis_bot/workflows/remote.py

```diff
@@ -53,14 +53,29 @@ def run(self, revision):
         dependencies = task['task']['dependencies']
         assert len(dependencies) > 0, 'No task dependencies to analyze'
 
-        # Find issues in dependencies
-        issues = []
-        for dep_id in dependencies:
+        # Skip dependencies not in group
+        # But log all skipped tasks
+        def _in_group(dep_id):
             if dep_id not in tasks:
                 # Used for docker images produced in tree
                 # and other artifacts
                 logger.info('Skip dependency not in group', task_id=dep_id)
-                continue
+                return False
+            return True
+        dependencies = list(filter(_in_group, dependencies))
+
+        # Do not run parsers when we only have a gecko decision task
+        # That means no analyzer were triggered by the taskgraph decision task
```

Comment:
I'd add "This can happen if the patch only touches file types for which we have no analyzer defined"

Reply:
(Nit)

Reply:
Done